src/GafferImageUI/ImageView.cpp

```diff
@@ -210,6 +230,8 @@ ImageView::ImageView( const std::string &name )
 	clampNode->minClampToPlug()->setValue( Color4f( 1.0f, 1.0f, 1.0f, 0.0f ) );
 	clampNode->maxClampToPlug()->setValue( Color4f( 0.0f, 0.0f, 0.0f, 1.0f ) );
 
+    addChild( new FloatPlug( "zoomLevel", Plug::In, 1.0f, 0.001, 100, Plug::Default & ~Plug::AcceptsInputs ) );
+
```

Comment:
I don't know of a strict limit, although I suppose you might hit floating point issues at the extremes? It seems more appropriate to have the limits be based on the useful range for viewing images rather than on the theoretical limits of OpenGL though I think.